torch/functional.py

```diff
@@ -433,20 +441,27 @@ def stft(input, n_fft, hop_length=None, win_length=None, window=None,
       :attr:`center` is ``True``. See :meth:`torch.nn.functional.pad` for
       all available options. Default is ``"reflect"``.
 
-    * If :attr:`onesided` is ``True`` (default), only values for :math:`\omega`
-      in :math:`\left[0, 1, 2, \dots, \left\lfloor \frac{\text{n\_fft}}{2} \right\rfloor + 1\right]`
-      are returned because the real-to-complex Fourier transform satisfies the
-      conjugate symmetry, i.e., :math:`X[m, \omega] = X[m, \text{n\_fft} - \omega]^*`.
+    * If :attr:`onesided` is ``True`` (default for real input), only values for
```

Comment:
Should there also be a warning that this behavior will change?

Reply:
The `onesided` default isn't deprecated but I added a warning to `return_complex`.

Reply:
Oh, sorry, this comment is at the wrong place. I also meant a warning in the documentation about the return_complex change.